client/app/license.txt

```diff
@@ -0,0 +1,755 @@
+-------------------------------------------------------------------------------
```

Comment:
File incorrectly added, to be removed.

Reply:
Still to be removed.

I'm going to fix this myself while integrating the pull request.